### ISSUE TITLE
Missing data

### ISSUE THREAD

**Issue by Anneli15** [chembl/IDG#57]
I can't see any of the data I have been adding manually. The compounds and added details are not in the Molecule dictionary and the targets not in the target dictionary.

**Comment by mnowotka** [comment]
Hi @revajk,

This is becoming more and more serous as I see. As you remember we have agreed that before you deploy any changes to production you will check if there are no issues.

Can you please check following facts:
- which modification of production server introduced this data loss?
- when did you upload this modification to the production?
- For how long production server was down?

Can you please follow the rules we agreed when deploying new changes to the production?

**Comment by revajk** [comment]
Anneli,  Is this all the data you have been adding ? Or something you did after Thursday?

**Comment by Anneli15** [comment]
This is data that I've added last week and maybe a bit the week before, before Thursday as I haven't used it since it was down at the end of last week (Thursday/Friday)

**Comment by Anneli15** [comment]
I think that you deplyed a previous version on Friday?

**Comment by Anneli15** [comment]
deployed...

**Comment by revajk** [comment]
No Michal said he didn’t ever it back to previous version.

**Comment by revajk** [comment]
revert it back  to previous version

**Comment by Anneli15** [comment]
Ok, hope you can work out what happened. I won't add anything more until you let me know it is fine, hopefully find the added data (new entries and added information to entries loaded by you) and you know what went wrong.

**Comment by revajk** [comment]
You have a record of how much data and list of molecules added?!

**Comment by Anneli15** [comment]
I save all my data still to an excel spreadsheet so i do have it. I have the compounds and information related to these. I don't have a precise record of the changes I've made to already loaded compounds and (targets, references etc) and target dictionary (I added some target complexes and families). I do have the data but not a list of compounds I made changes to as such. And not a list of target components and descriptions, took them from chembl as I was adding them.
